### DISCUSSION TITLE
关于页在about的index.md定义后的显示问题

### DISCUSSION THREAD

**Discussion by neokoni** [MaLuns/hexo-theme-async#93] [Q&A 疑难杂症] [answered]
刚刚开始配置主题
#### 在about/index.md定义后
![20231003110309_1](https://github.com/MaLuns/hexo-theme-async/assets/90206655/0e79b6b7-633b-4807-b0a7-e11af2e2982f)
#### 原版
![20231003110633_1](https://github.com/MaLuns/hexo-theme-async/assets/90206655/781c9d3f-7fa2-4ada-b0d0-86811bcb096c)
有一个md内容的显示问题。位置不对？希望有个解决的办法

**Comment by MaLuns** [comment, marked as answer]
md 和配置的两个现在是不能共存的，后面优化做下兼容处理
